consensus/blake3pow/consensus.go

```diff
@@ -226,6 +226,9 @@ func (blake3pow *Blake3pow) VerifyUncles(chain consensus.ChainReader, block *typ
 		if err != nil {
 			return err
 		}
+		if uncle.NumberU64() < 2*params.BlocksPerMonth && uncle.Lock() != 0 {
```

Comment:
Shouldn't you specify `ZONE_CTX` when getting the number here? IIUC `BlocksPerMonth` is a count of zone blocks

Reply:
since uncle is work object header, it only has zone number